app/views/hosts/_form.html.erb

```diff
@@ -22,6 +23,10 @@
       <% end %>
       <li><a href="#params" id='params-tab' data-url='<%= current_parameters_hosts_path %>' data-url2='<%= puppetclass_parameters_hosts_path %>' data-toggle="tab"><%= _('Parameters') %></a></li>
       <li><a href="#info" data-toggle="tab"><%= _('Additional Information') %></a></li>
+
+      <% @tabs.keys.each do |tab| %>
```

Comment:
Current tabs/properties would be sorted out too of course. I have already shown this for puppet configuration in PR #2555. In the [google groups thread](https://groups.google.com/forum/#!searchin/foreman-dev/host$20unification/foreman-dev/esDULZDlOK8/rmR-Rm9L3kIJ) I have mentioned all aspects that I can see in the near future.
This PR is about adding core support for aspects only. Once it gets merged, I will start working on moving puppet to an aspect.